### PR TITLE
UIOR-1212: Bump minimum okapi interfaces for `pieces` (2.0 3.0) and `receiving` (1.0 2.0)

### PR DESCRIPTION
**Pull request by alisher-epam**
## Purpose

[UIOR-1212](https://issues.folio.org/browse/UIOR-1212): Bump minimum okapi interfaces for `pieces` (3.0) and `receiving` (2.0)
## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [x] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

**Comment by alisher-epam** [comment]
Other related PRS
- https://github.com/folio-org/ui-invoice/pull/754
- https://github.com/folio-org/ui-receiving/pull/488
- https://github.com/folio-org/ui-inventory/pull/2395

**Review by usavkov-epam — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -50,9 +50,9 @@
       "orders.export-history": "1.0",
       "organizations.organizations": "1.0",
       "organizations-storage.organization-types": "1.0",
-      "pieces": "2.0",
+      "pieces": "3.0",
       "po-number": "1.0",
-      "receiving": "1.0",
+      "receiving": "2.0",
```
Comment (NikitaSedyx) [inline package.json:55]: same here

**Comment by NikitaSedyx** [comment]
are updated fields used in ui-orders?

File: package.json [hunk 2]
```diff
@@ -50,9 +50,9 @@
       "orders.export-history": "1.0",
       "organizations.organizations": "1.0",
       "organizations-storage.organization-types": "1.0",
-      "pieces": "2.0",
+      "pieces": "3.0",
```
[the remark this answers is not in the document]
Comment (alisher-epam) [reply package.json:53]: Sorry not aware of that format although I have seen such formatting in the package.json. I will take a look it again.

[the remark this answers is not in the document]
Comment (alisher-epam) [reply package.json:53]: @SerhiiNosko does the backend support both versions for the changes here?

[the remark this answers is not in the document]
Comment (NikitaSedyx) [reply package.json:53]: it's not BE related, it's question about UI, module supports 2.0 and 3.0 or just 3.0

[the remark this answers is not in the document]
Comment (alisher-epam) [reply package.json]: You are right. After discussion with Serhii, your suggestion is totally make sense. And updated PR accordingly

File: package.json [hunk 3]
```diff
@@ -50,9 +50,9 @@
       "orders.export-history": "1.0",
       "organizations.organizations": "1.0",
       "organizations-storage.organization-types": "1.0",
-      "pieces": "2.0",
+      "pieces": "2.0 3.0",
       "po-number": "1.0",
-      "receiving": "1.0",
+      "receiving": "1.0 2.0 ",
```
Comment (SerhiiNosko) [inline package.json:55]: 1 space to delete

Comment (SerhiiNosko) [reply package.json]: fixed

**Review by SerhiiNosko — APPROVED** [review]

**Comment by usavkov-epam** [comment]
Major version update is introduced at https://github.com/folio-org/ui-orders/pull/1536

**Comment by zburke** [comment]
@alisher-epam, the description of the change ("Bump minimum okapi interfaces") and the actual change ("add support for additional okapi interface versions") are different here! The change is fine, but it's out of sync with the comment in git and the CHANGELOG. You don't need to change anything here, just try to be attentive to this kind of detail in future work. Thanks!

**Comment by alisher-epam** [comment]
@zburke Thank you for your advice. I will keep it in my mind.

**Pull request merged**